### PR TITLE
Resolve notebook paths

### LINKED ISSUE #261: Run the kernel in the same directory as the notebook
If you open a notebook from nteract, it launches the kernel process with the current working directory to where nteract was launched from. Child processes need the path to be set properly.

### PR DESCRIPTION
**Pull request by jdfreder**
closes #261

### CODE REVIEW

**Comment by rgbkrk** [comment]
There we go! Now @Carreau's awesome OS X stuff handles the cli opening of a notebook well.

File: src/main/index.js [hunk 1]
```diff
@@ -24,5 +25,5 @@ app.on('ready', () => {
   Menu.setApplicationMenu(defaultMenu);
   // Let the kernels/languages come in after
   loadFullMenu().then(menu => Menu.setApplicationMenu(menu));
-  notebooks.forEach(launchFilename);
+  notebooks.map(f=>resolve(f)).forEach(launchFilename);
```
Comment (rgbkrk) [inline src/main/index.js:28]: Could turn this into:

``` js
notebooks.forEach(f => {
  launchFilename(resolve(f))
})
```

Comment (Carreau) [inline src/main/index.js:28]: or `notebooks.map(resolve).forEach(launchFilename);` ?

Comment (jdfreder) [inline src/main/index.js:28]: You know what I like even more, `.map(resolve).foreach(launchFilename)`

Comment (rgbkrk) [inline src/main/index.js:28]: that is very readable

**Pull request merged**